src/index.html

```diff
@@ -1,12 +1,47 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Calendar</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+
+  <body>
+    <div class="calendar calendar--sun calendar--month-length--31">
+      <div class="day day--number--1"></div>

```

Comment:
```suggestion
      <div class="calendar__day"></div>

```
Тебе не нужен тут дополнительный класс